integrations/istio/prometheus_metadata.yaml

```diff
@@ -8,6 +8,22 @@ platforms:
       name: Istio Prometheus Exporter
       doc_url: https://istio.io/latest/docs/ops/integrations/prometheus
     default_metrics:
+      - name: prometheus.googleapis.com/istio_requests_total/counter
+        prometheus_name: istio_requests_total
+        kind: CUMULATIVE
+        value_type: DOUBLE
+      - name: prometheus.googleapis.com/istio_request_duration_milliseconds/histogram
+        prometheus_name: istio_request_duration_milliseconds
+        kind: CUMULATIVE
+        value_type: DISTRIBUTION
+      - name: prometheus.googleapis.com/istio_request_bytes/histogram
+        prometheus_name: istio_request_bytes
+        kind: CUMULATIVE
+        value_type: DISTRIBUTION
+      - name: prometheus.googleapis.com/istio_response_bytes/histogram
+        prometheus_name: istio_response_bytes
+        kind: CUMULATIVE
+        value_type: DISTRIBUTION
```

Comment:
Just double-checking if intentional... the dashboard doesn't showcase `istio_request_bytes` and `istio_response_bytes`

Reply:
Yes, not featured in the dashboard, but these are collected by the integration.